network-api/networkapi/campaign/views.py

```diff
@@ -215,11 +218,7 @@ def petition_submission(request, petition):
     message = json.dumps({
         'app': settings.HEROKU_APP_NAME,
         'timestamp': datetime.now().isoformat(),
-        'data': {
```

Comment:
Please see comment above

Reply:
Looks good, but can you please confirm with Shawn that the app and timestamp fields are fine being outside of the data object?